### ISSUE TITLE
Updating to work with Cypress v7.x

### ISSUE THREAD

**Issue by philga7** [dmtrKovalenko/cypress-real-events#44]
Hi:

We are currently beholden to Cypress `v6.9.1` and cannot upgrade to the latest `v7.x` because of the following.

When running node `v14.15.4`, an error message is produced by `cypress-real-events` stating the following (in Node `v12.21`, this is a warning, not an error message):

```
npm ERR! Could not resolve dependency:
npm ERR! peer cypress@"^4.x || ^5.x || ^6.x" from cypress-real-events@1.3.0
npm ERR! node_modules/cypress-real-events
npm ERR!   dev cypress-real-events@"^1.3.0" from the root project
```

I haven't taken a look at `cypress-real-events` code to see why this version threshold exists; is there a reason why? Then, based off of that answer, what would it take to become compatible with Cypress `v.7.x`?

Thanks!

**Comment by dmtrKovalenko** [comment]
I think this is just outdated dependencies list. I will try to fix this and publish new version today

**Comment by philga7** [comment]
Sweet! Thanks.

**Comment by avallete** [comment]
In the mean time for anyone wanting to bypass this issue:

```bash
# Install the dependencies without peerDependencies version mismatch errors
npm install -D --force
# Once the package-lock has been generated you should not have the error anymore
npm install -D
# or
npm ci
```

**Comment by silversonicaxel** [comment]
are there updates regarding this issue?

**Comment by dmtrKovalenko** [comment]
Released in 1.4.0
